4.analyze_data/notebooks/linear_model/0.fit_linear_model_strat.ipynb

```diff
@@ -19,7 +19,7 @@
     "from sklearn.linear_model import LinearRegression\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    # Check if any rows match the undesired combinations
</pre><p>Similar comment about the <code>undesired_rows</code> commont  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/CFReT_data/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>